scss/partials/_blog.scss

```diff
@@ -0,0 +1,404 @@
+        }
+
+        p {
+            color: $cool-neutral-1;
+
+            .dropcap {
+                @extend .tk-degular;
+                color: $accessible-green;
+                float: left;
+                font-size: 16rem;
+                font-weight: 900;
+                line-height: 0.35;
+                margin: 0.1em 0.1em 0.18em 0;
+            }
+        }
+
+        &-blockquote {
+            &--quote {
+                position: relative;
+                margin-top: 0;
+                margin-bottom: 1.9rem;
+
+                &:before {
+                    content: "";
+                    display: block;
+                    border-top: solid 0.1rem $cool-neutral-6;
+                    width: 100%;
+                    height: 0.1rem;
+                    position: absolute;
+                    top: 50%;
+                    z-index: 1;
+                }
+
+                &-background {
+                    background-image: url("data:image/svg+xml,%3Csvg width='20' height='16' viewBox='0 0 20 16' fill='none' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='M11.3181 9.294C11.3181 13.116 13.2921 15.636 16.2741 15.636C18.4161 15.636 19.8021 14.25 19.8021 12.024C19.8021 10.722 19.4661 10.176 18.8781 9.588C17.5761 8.286 14.8041 9.504 14.8041 6.984C14.8041 4.758 17.0721 3.078 19.1721 3.078C19.5501 3.078 19.7601 2.868 19.7601 2.49V1.44C19.7601 1.02 19.5081 0.767998 18.8781 0.767998C15.4761 0.767998 11.3181 4.128 11.3181 9.294ZM0.188078 9.294C0.188078 13.116 2.16208 15.636 5.14408 15.636C7.28608 15.636 8.67208 14.25 8.67208 12.024C8.67208 10.722 8.33608 10.176 7.74808 9.588C6.44608 8.286 3.67408 9.504 3.67408 6.984C3.67408 4.758 5.94208 3.078 8.04208 3.078C8.42008 3.078 8.63008 2.868 8.63008 2.49V1.44C8.63008 1.02 8.37808 0.767998 7.74808 0.767998C4.34608 0.767998 0.188078 4.128 0.188078 9.294Z' fill='%23509E2F'/%3E%3C/svg%3E%0A");
+                    background-position: center;
+                    background-repeat: no-repeat;
+
+                    height: 2rem;
+                    width: 2rem;
+                    background-color: #fff;
+                    padding: 0 3.4rem;
+                    position: relative;
+                    z-index: 5;
+                }
+            }
+
+
+            @extend .tk-degular-display;
+            @extend .h5;
+            border-bottom: 0.1rem solid $cool-neutral-6;
+            margin-bottom: 11.6rem;
+            text-align: center;
+            padding-bottom: 4rem;
+
+            &-cite {
+                font-weight: 900;
+                font-style: normal;
+                display: block;
+                padding-top: 25px;
+            }
+
+            &-span {
+                @extend .tk-aktiv-grotesk;
+                display: block;
+                font-size: 14px;
+                color: $secondary-green-1;
+                letter-spacing: 3.2px;
+                padding-top: 0.8rem;
+            }
+        }
+
+        &-aside {
```

Comment:
Like you did above, I'd like to see us break BEM here for content simplicity and just have this apply to any default aside in a blog post (we can deal with variants later if they come up).

Reply:
Done. Did the same for blockquotes.